src/cdk/drag-drop/drop.ts

```diff
@@ -369,4 +383,18 @@ export class CdkDrop<T = any> implements OnInit, OnDestroy {
           yOffset >= Math.floor(clientRect.top) && yOffset <= Math.floor(clientRect.bottom);
     });
   }
+
+  /**
+   * Checks whether the pointer coordinates are close to the drop container.
+   * @param xOffset Coordinates along the X axis.
+   * @param yOffset Coordinates along the Y axis.
+   */
+  private _isPointerNearDropContainer(xOffset: number, yOffset: number): boolean {
```

Comment:
I snuck in a suggestion w/ the function name comment on the parameter names too: `pointerX` and `pointerY`

Reply:
I went with `xOffset` and `yOffset`, because it's what we call them in all the other functions.

Reply:
I think it would be good to rename them in a follow-up PR, then, if they all do represent the pointer position.

Reply:
Done.